### PR TITLE
[Word] (list) Add manage-style snippet (preview)

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

File: samples/word/99-preview-apis/manage-list-styles.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+id: word-lists-manage-styles
+name: Manage list styles
```
Comment (AlexJerabek) [inline samples/word/99-preview-apis/manage-list-styles.yaml:2]: "Manage" seems to oversell the API capabilities. Consider "Get" instead.
```suggestion
name: Get list styles
```

File: samples/word/99-preview-apis/manage-list-styles.yaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    content: |
+        $("#count").on("click", () => tryCatch(getCount));
+        $("#properties").on("click", () => tryCatch(getProperties));
+
+        async function getCount() {
+          // Gets the available list styles stored with the document.
+          await Word.run(async (context) => {
+            const styles: Word.StyleCollection = context.document.getStyles();
+            const count = styles.getCount();
+            styles.load();
```
Comment (AlexJerabek) [inline samples/word/99-preview-apis/manage-list-styles.yaml:18]: Here and elsewhere, let's avoid loading more properties than necessary to show best practices.

File: samples/word/99-preview-apis/manage-list-styles.yaml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+            for (let i = 0; i <= count.value; i++) {
+              if (styles.items[i] && styles.items[i].type == "List") {
+                console.log(`List style name: ${styles.items[i].nameLocal}`, styles.items[i]);
+              }
+            }
+          });
+        }
+
+        async function getProperties() {
```
Comment (AlexJerabek) [inline samples/word/99-preview-apis/manage-list-styles.yaml:29]: The only properties shown are the list levels. Can we rename this function (and the buttons and comments) to reflect that?

**Review by AlexJerabek — CHANGES_REQUESTED** [review]
Looks good! Just a couple comments, though if you take them, I'd like to double-check things before merging.

File: samples/word/99-preview-apis/manage-list-styles.yaml (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply samples/word/99-preview-apis/manage-list-styles.yaml:2]: @AlexJerabek Yeah, I have some further changes planned but I can rename (and update descriptions, etc.) accordingly when that time comes.

File: samples/word/99-preview-apis/manage-list-styles.yaml (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply samples/word/99-preview-apis/manage-list-styles.yaml:18]: I'm loading everything for now so users can see all the properties in the log. Maybe I'll add a code comment to that effect, which will explain why we're not strictly following best practice.

File: samples/word/99-preview-apis/manage-list-styles.yaml (hunk 3 above)
Comment (ElizabethSamuel-MSFT) [reply samples/word/99-preview-apis/manage-list-styles.yaml:29]: Right now, we're logging all the properties of the list style along with the list levels so the name, etc. is appropriate.

**Review by AlexJerabek — APPROVED** [review]

**Pull request merged**